pkg/network/render.go

```diff
@@ -338,13 +339,15 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	case len(prev.ServiceNetwork) > len(next.ServiceNetwork):
 		// Going from dual to single
 		dualStack, singleStack = prev, next
+	case !reflect.DeepEqual(prev.ServiceNetwork, next.ServiceNetwork):
+		return errors.Errorf("unsupported change to ServiceNetwork")
+	case isClusterNetworkChangeSafe(prev, next):
+		// this was a CIDR Expansion change not a single/dual stack change. can just get out of here now.
```

Comment:
If `isClusterNetworkChangeSafe` changes to include more/other verification this comment would no longer be valid